### ISSUE TITLE
[Android] Initialize failed with unauthorized access exception.

### ISSUE THREAD

**Issue by MinRam** [gwiazdorrr/BetterStreamingAssets#36]
- **Platform**:
  -  **Unity**： Unity 2021.3.11f1 Personal - Window 11
  - **Visual Studio**: Microsoft Visual Studio Enterprise 2022 (64-bit) - LTSC 17.2
  - **Android Phone**: Mi 10s

- **Code**:
```c#
BetterStreamingAssets.Initialize();
```

- **Exception**:
```
System.UnauthorizedAccessException: 
Access to the path \'/data/data/{appName}/pram-shadow-files\' is denied.\n
```

I found the code of <code>GetStreamingAssetsInfoFromJar</code> with <code>File.OpenRead</code> in `ApkImp`.
<img width="893" alt="image" src="https://user-images.githubusercontent.com/14797464/210223163-9d97b924-c414-44a0-a2aa-f8ebe1ea7a06.png">

**Comment by gwiazdorrr** [comment]
This seems like an issue with patching. I'll have a look. In the meantime, you'll have to use the full build.

**Comment by ArgouHL** [comment]
Any Update that can use BSA while develop with patching to my andriod?

**Comment by gwiazdorrr** [comment]
Closed with dea9e36c78140cbd39d783b41a1e2a363175d3e9. Tests look good here, please raise this issue again if the workaround does not work for you.
